src/test/base32_tests.cpp

```diff
@@ -1,20 +1,38 @@
+// Copyright (c) 2012-2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <util/strencodings.h>
+
 #include <boost/test/unit_test.hpp>
+#include <string>
 
-#include "util.h"
+using namespace std::literals;
 
 BOOST_AUTO_TEST_SUITE(base32_tests)
 
 BOOST_AUTO_TEST_CASE(base32_testvectors)
 {
     static const std::string vstrIn[]  = {"","f","fo","foo","foob","fooba","foobar"};
     static const std::string vstrOut[] = {"","my======","mzxq====","mzxw6===","mzxw6yq=","mzxw6ytb","mzxw6ytboi======"};
-    for (unsigned int i = 0; i < std::size(vstrIn); i++)
+    for (unsigned int i=0; i<std::size(vstrIn); i++)
```

Comment:
The original formatting was correct. Not the above.

Reply:
https://github.com/barton2526/Gridcoin-Research/commit/f9760d52b11de6f1924e1cb4403bba07b30a7484